scene/gui/container.h

```diff
@@ -42,6 +42,8 @@ class Container : public Control {
 
 protected:
 	void queue_sort();
+	Control *is_sortable_control(Node *p_node) const;
```

Comment:
```suggestion
	Control *as_sortable_control(Node *p_node) const;
```
Or `get`, `is` doesn't work logically IMO